### ISSUE TITLE
Error on launch

### ISSUE THREAD

**Issue by mosuem** [juliansteenbakker/mobile_scanner#70]
I cannot launch this due to

```
e: C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.0.0\android\src\main\kotlin\dev\steenbakker\mobile_scanner\MobileScanner.kt: (29, 1): Class 'MobileScanner' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(p0: Int, p1: Array<(out) String!
>, p2: IntArray): Boolean defined in io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener
e: C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.0.0\android\src\main\kotlin\dev\steenbakker\mobile_scanner\MobileScanner.kt: (69, 5): 'onRequestPermissionsResult' overrides nothing
e: C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.0.0\android\src\main\kotlin\dev\steenbakker\mobile_scanner\MobileScanner.kt: (70, 66): Type mismatch: inferred type is Array<CapturedType(out String)>? but Array<(out) String!> was expected
e: C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.0.0\android\src\main\kotlin\dev\steenbakker\mobile_scanner\MobileScanner.kt: (70, 79): Type mismatch: inferred type is IntArray? but IntArray was expected
```

Changing the lines to the proposed ones lets me run it. 

Thanks so much for this great plugin!

**Comment by mosuem** [comment]
(Also, there are two warnings
```
/C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/mobile_scanner-1.0.0/lib/src/mobile_scanner.dart:50:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/src/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addObserver(this);
                   ^
/C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/mobile_scanner-1.0.0/lib/src/mobile_scanner.dart:131:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/src/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.removeObserver(this);
```)

**Comment by mosuem** [comment]
I see this is to be fixed in pull request #63! Thanks.
